include/simdjson/inline/parsedjson_iterator.h

```diff
@@ -208,9 +208,17 @@ bool dom::parser::Iterator::next() {
   current_type = next_type;
   return true;
 }
-
+/// Os3 2
 dom::parser::Iterator::Iterator(const dom::parser &pj) noexcept(false)
```

Comment:
Please remove these changes to the constructor. It will handled in a distinct PR. Please remove the /// Os3 2 comment.

Reply:
done